merlin/examples/workflows/hello/hello_soft_fail.yaml

```diff
@@ -0,0 +1,24 @@
+description:
```

Comment:
Can we make a version that is based off hello_samples too? That will test the $(MERLIN_SAMPLE_PATH) hierarchy

Reply:
Or maybe a test based off hello_samples?

Reply:
Yeah sure I can add a version for hello_samples.